### ISSUE TITLE
Hi! Any chance to get this app ready with SDK3 for Homey Pro 2023? ℹ️

### ISSUE THREAD

**Issue by corvdniet** [Dijker/nu.dijker.PaperTrails#52]
**What is your Homey version (Settings - System), your App version (Settings-Apps) ?**

Homey x.x.x
PaperTrails 0.x.x
Send Log to Developer: xxx-xxxx-xxxx

**What did you try to achieve?**
*Please include detailed steps how we can reproduce the issue*

**What was the result?**
...

**What did you expect as result?**

...

**Any other remarks**

...

**Comment by hjelmdal** [comment]
Bump and +1 :)

**Comment by ppheijnen** [comment]
Bump +1

**Comment by JohPePe** [comment]
Bump +1

**Comment by damo78** [comment]
Keep on bumping, lift your issue high, everybody bumping until you see it done in your eye(s) +1

**Comment by gandalfsn** [comment]
Bump +1

**Comment by betsim** [comment]
+1

**Comment by rcoemans** [comment]
I am using PaperTrails app in almost all my flows and so this for me is quite a blocker for upgrading to Homey Pro (Early 2023). Hopefully this app can be made compatible for the latest Homey Pro.

**Comment by rpb006** [comment]
Yes, will be very nice if this app is updated for the new 2023 pro version
